client/components/Cohorts/CohortScenarios.jsx

```diff
@@ -275,312 +254,219 @@ export class CohortScenarios extends React.Component {
     );
 
     return (
-      <Container fluid className="c__table-container">
-        {editorMenu}
```

Comment:
@enatario is the `editorMenu` meant to be removed? Because it has been removed, the search bar is no longer appearing.

Reply:
Ah yes! I suppose I should move this back to draft for now since it'll rely on a few other items first.

The search would come up in a modal if you want to edit the scenario list and what should be shown by default are the ones chosen via the cohort creation process.